scripts/xtensa-build-zephyr.py

```diff
@@ -23,6 +23,9 @@
 west_top = pathlib.Path(SOF_TOP, "zephyrproject")
 default_rimage_key = pathlib.Path("modules", "audio", "sof", "keys", "otc_private_key.pem")
 
+# Initial value of SOF firmware file version
+sof_version = "0.0.0"
```

Comment:
```suggestion
sof_version = None
```

Reply:
Updated. Set sof_version is initialized to None now.